        syntax = "proto3";
 
	message SearchRequest
	{
		string Request = 1;
		string strUser = 2;
		string strPasswd = 3;
		string strLTime = 4;
		string strMode = 5;
	}
 
	message SearchResponse
	{
		string Response = 1;
	}
 
	service SearchService {
  		rpc Search (SearchRequest) returns (SearchResponse);
	}
